mbrl/util/mujoco.py

```diff
@@ -217,9 +219,20 @@ def __init__(self, env: gym.wrappers.TimeLimit):
         elif "mbrl.third_party.dmc2gym" in self._env.env.__class__.__module__:
             self._enter_method = self._enter_dmcontrol
             self._exit_method = self._exit_dmcontrol
+        elif "gym.envs.robotics" in self._env.env.__class__.__module__:
```

Comment:
I'm observing some weird behavior where the evaluation function starts returning trajectories with all 0 rewards, yet this is not reflected in the actual evaluated trajectory. Have you seen this? Seems to be related to having multiple processes, maybe the environments are sharing some resource?

Reply:
Thanks! That would explain a lot. I'll have to look into that.